Matey/Controllers/AddNewTransactionViewController.swift

```diff
@@ -0,0 +1,109 @@
+
+    func uiInit() {
+        title = ConstantsAddNewTransactionVC.title
+        navigationController?.navigationBar.tintColor = ConstantsAddNewTransactionVC.navigationBarTintColor
+
+        textFieldRegistrantsName.placeholder = ConstantsAddNewTransactionVC.textFieldRegistrantsNamePlaceholder
+        textFieldRegistrantsName.textFieldCornerRadius()
+        textFieldRegistrantsName.textFieldLeftPadding()
+
+        textFieldRegistrantsUsername.placeholder = ConstantsAddNewTransactionVC.textFieldRegistrantsUsernamePlaceholder
+        textFieldRegistrantsUsername.textFieldCornerRadius()
+        textFieldRegistrantsUsername.textFieldLeftPadding()
+
+        textFieldFriendUsername.placeholder = ConstantsAddNewTransactionVC.textFieldFriendUsernamePlaceholder
+        textFieldFriendUsername.textFieldCornerRadius()
+        textFieldFriendUsername.textFieldLeftPadding()
+
+        textFieldLendAmount.placeholder = ConstantsAddNewTransactionVC.textFieldLendAmountPlaceholder
+        textFieldLendAmount.keyboardType = ConstantsAddNewTransactionVC.textFieldLendAmountKeyboardType
+        textFieldLendAmount.textFieldCornerRadius()
+        textFieldLendAmount.textFieldLeftPadding()
+
+        textFieldBorrowAmount.placeholder = ConstantsAddNewTransactionVC.textFieldBorrowAmountPlaceholder
+        textFieldBorrowAmount.keyboardType = ConstantsAddNewTransactionVC.textFieldBorrowAmountKeyboardType
+        textFieldBorrowAmount.textFieldCornerRadius()
+        textFieldBorrowAmount.textFieldLeftPadding()
+
+        buttonSaveNewTransaction.backgroundColor = ConstantsAddNewTransactionVC.buttonSaveNewTransactionBackgroundColor
+        buttonSaveNewTransaction.tintColor = ConstantsAddNewTransactionVC.buttonSaveNewTransactionTintColor
+        buttonSaveNewTransaction.layer.cornerRadius = ConstantsAddNewTransactionVC.buttonSaveNewTransactionCornerRadius
+        buttonSaveNewTransaction.setTitle(ConstantsAddNewTransactionVC.buttonSaveNewTransactionTitle, for: .normal)
+    }
+
+    func showAlert(title: String, message: String) {
+        let alertController = UIAlertController(title: title, message: message, preferredStyle: .alert)
+        let doneAction = UIAlertAction(title: ConstantsAddNewTransactionVC.messageDone, style: UIAlertAction.Style.default) { (action) in }
+
+        alertController.addAction(doneAction)
+        alertController.view.tintColor = UIColor.black
+        self.present(alertController, animated: true, completion: nil)
+    }
+
+    func saveTransaction() {
+        let textRegistrantsName = textFieldRegistrantsName.text ?? ""
+        let textFieldRegistrantsUsername = textFieldRegistrantsUsername.text ?? ""
+        let textFieldFriendUsername = textFieldFriendUsername.text ?? ""
+
+        if viewModel?.validationTextFields(textRegistrantsName: textRegistrantsName, textFieldRegistrantsUsername: textFieldRegistrantsUsername, textFieldFriendUsername: textFieldFriendUsername) == true {
```

Comment:
```suggestion
        if viewModel?.validationTextFields(textRegistrantsName: textRegistrantsName, textFieldRegistrantsUsername: textFieldRegistrantsUsername, textFieldFriendUsername: textFieldFriendUsername) {
```
do you think this is necessary?

Reply:
With optional booleans it's needed to make the check explicit